pkg/operator/api/context/apis.go

```diff
@@ -27,6 +27,7 @@ type API struct {
 	*ComputedResourceFields
 	Path                  string  `json:"path"`
 	RequestHandlerImplKey *string `json:"request_handler_impl_key"`
+	MetadataKey           string  `json:"metadata_key"`
```

Comment:
I think we should use something other than metadata for this purpose since metadata is assumed to be a single JSON file (see `get_metadata_url()`, which is called in `context.get_metadata()` and `context.write_metadata()`. Maybe `classesKey`?

Reply:
I thought this would be deleted as part of removing e2e

Reply:
Doesn't look like it is being used anywhere, can I delete it?

Reply:
Let's discuss at standup so Ivan can weigh in. Either we will remove, leave this PR as is, and change the definition of metadata, or we will change this PR to use a different path (e.g. `classes`, `classKeys`, `classesCache`, ...) and leave metadata as is (or remove it anyways).